Samples/android/Android-SDK-Demo-App/app-v7-java/build.gradle

```diff
@@ -19,8 +19,10 @@ android {
 
     buildTypes {
         release {
-            minifyEnabled false
+            minifyEnabled true
+            shrinkResources true
             proguardFiles getDefaultProguardFile('proguard-android-optimize.txt'), 'proguard-rules.pro'
+            signingConfig signingConfigs.debug
```

Comment:
no signingConfig found in this gradle, we can copy signingConfig from kotlin project and use release instead of debug